unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1502,90 +1504,6 @@ inputDescription input =
       -- just trying to recover the syntax the user wrote
       These path _branch -> pure (Path.toText' path)
 
-handleStructuredFindI :: HQ.HashQualified Name -> Cli ()
```

Comment:
Moved the find/replace stuff to its own module file since I add to add a new helper for rendering files and we're still trying to split up HandleInput when we can :)